src/components/WalletSuite/WalletSelectorOpener/Connector.tsx

```diff
@@ -22,26 +21,10 @@ export default function Connector(props: Connection) {
       className="flex flex-col items-center justify-center gap-1 h-28 p-5 border border-gray-l2 rounded bg-white hover:bg-orange-l5 dark:bg-blue/50 hover:dark:bg-blue-d3 dark:border-none"
       onClick={handleConnect}
     >
-      <Logo logo={props.logo} />
+      <Image src={props.logo} className={"w-12 h-12"} />
```

Comment:
nit: no need for surrounding curly brackets, `...={"..."}` can be just `...="..."`